src/containers/ViewerPanel/style.css

```
@@ -10,14 +10,11 @@
     gap: 4px;
     bottom: 0;
     margin-bottom: 20px;
-}
-
-.default-control-spacing {
     right: 25%;
     width: 50%;
 }
 
 .full-width-bottom-controls {
     right: 5%;
     left: 5%;
```

```suggestion
    left: 5%;
    width: 90%;
```

I like this change, but we also need to override the width property in this case.